static/docs/commands-reference/status.md

```diff
@@ -56,7 +56,12 @@ differ are listed. For each item listed, either the file name or the checksum is
 shown, and additionally a status word is shown describing the change:
 
 - For the local workspace:
-  - _changed_ means the named file has changed
+  - _changed deps_ means the named file has changed dependencies
+  - _changed outs_ means the named file has changed outputs
+  - _changed checksum_ means actual check sum of data dosen't match specified in
+    dvc files
+  - _always checksum_ means special dvc file with no dependencies, considered
```

Comment:
DVC-file

Reply:
is it `always checksum` or something else? cc @efiop

Reply:
It is something else, for sure. Probably `always changed`.